src/cli/api/local_api.py

```diff
@@ -36,24 +54,140 @@ def __is_config_valid(self, application: str) -> bool:
         pass
 
     def get_application_config(self, application: str) -> AppConfigType:
-        pass
+        app_details = self.__config_manager.get_application(application)
+
+        app_config_path = Path(app_details['path']) / 'config'
+        application_json_path = app_config_path / 'application.json'
+        network_json_path = app_config_path / 'network.json'
+
+        application_data = read_json_file(application_json_path)
+        network_data = read_json_file(network_json_path)
+
+        app_config = {"application": application_data, "network": network_data}
+        return app_config
 
     def create_experiment(
-        self, name: str, app_config: AppConfigType, local: bool
-    ) -> None:
-        pass
+        self, name: str, app_config: AppConfigType, network: str, path: Path, application: str
+    ) -> Tuple[bool, str]:
 
-    def delete_experiment(self, path: str) -> None:
-        pass
+        experiment_directory = path / name
+        if experiment_directory.is_dir():
+            return False, f'Experiment directory {name} already exists.'
+
+        experiment_directory.mkdir(parents=True)
+
+        input_directory = experiment_directory / 'input'
+        input_directory.mkdir(parents=True)
+        self.__copy_input_files_from_application(application, input_directory)
+
+        experiment_json_file = experiment_directory / 'experiment.json'
+        experiment_meta = {
+            "backend": {
+                "location": "local",
+                "type": "local_netsquid"
+             },
+            "number_of_rounds": 1,
+            "description": f"{name}: experiment description"
+        }
+
+        asset_application = self.__create_asset_application(app_config)
```

Comment:
As said split creation of asset (and later when you run the experiment roundset)